.github/workflows/pr-review-advisor.yaml

```
@@ -328,6 +347,100 @@ jobs:
+    timeout-minutes: 40
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        advisor:
+          - { interest: behavior, label: Behavior, model: azure/openai/gpt-5.6-terra, sandbox_name: pr-advisor-sp-behavior, artifact_dir: pr-review-specialist-behavior, artifact_name: pr-review-specialist-behavior }
+          - { interest: trust, label: Trust, model: azure/openai/gpt-5.6-terra, sandbox_name: pr-advisor-sp-trust, artifact_dir: pr-review-specialist-trust, artifact_name: pr-review-specialist-trust }
+          - { interest: design-architecture, label: Design / Architecture, model: azure/openai/gpt-5.6-terra, sandbox_name: pr-advisor-sp-design, artifact_dir: pr-review-specialist-design-architecture, artifact_name: pr-review-specialist-design-architecture }
+          - { interest: operations, label: Operations, model: azure/openai/gpt-5.6-terra, sandbox_name: pr-advisor-sp-operations, artifact_dir: pr-review-specialist-operations, artifact_name: pr-review-specialist-operations }
+          - { interest: documentation, label: Documentation, model: azure/openai/gpt-5.6-terra, sandbox_name: pr-advisor-sp-docs, artifact_dir: pr-review-specialist-documentation, artifact_name: pr-review-specialist-documentation }
+    env: &shadow-analysis-env
```

actionlint (1.7.12)

[error] anchor "shadow-analysis-env" is defined but not used

(syntax-check)